### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,4 +2,5 @@ version=2.6.1-SNAPSHOT
 org.gradle.caching=true
 org.gradle.parallel=true
 kafkaVersion=3.3.1
-confluentVersion=7.3.0
+confluentVersion=7.3.1
+org.gradle.jvmargs=-Xmx2048m
```
Comment (yannick-roeder) [inline gradle.properties:6]: Whats the reason for setting this jvmarg here as a default?

Comment (philipp94831) [reply gradle.properties:6]: Sonarqube crashed with the Gradle update

File: build.gradle.kts [hunk 2]
```diff
@@ -133,3 +133,9 @@ configure<org.hildan.github.changelog.plugin.GitHubChangelogExtension> {
 tasks.withType<Test> {
     useJUnitPlatform()
 }
+
+release {
+    git {
+        requireBranch.set("master")
```
Comment (yannick-roeder) [inline build.gradle.kts:139]: What about `main` branches as default branch?

Comment (philipp94831) [reply build.gradle.kts:139]: This repo uses master

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**